test/integration_test.py

```diff
@@ -60,7 +60,20 @@ def main():
         assert view[1][1] == 'e'
 
         worksheet.refresh()
-        assert worksheet[3][4] == 'e'
+        assert worksheet.view()[3][4] == 'e'
```

Comment:
⚠️ Is this change necessary? If it's only for consistency inside this test then no problemo 👌 But if it's necessary (I don't think so from the code changes) it could break existing code, can it be avoided?

Reply:
Good point. The integration test actually was broken since 52babe2c996728a4f311e8a6c13e6364d6abc4f0, this is just making it work again.